homeassistant/components/tibber/__init__.py

```diff
@@ -82,7 +62,7 @@ async def _close(event):
     # have to use discovery to load platform.
     hass.async_create_task(
         discovery.async_load_platform(
-            hass, "notify", DOMAIN, {CONF_NAME: DOMAIN}, hass.data[DATA_HASS_CONFIG]
+            hass, "notify", DOMAIN, {}, hass.data[DATA_HASS_CONFIG]
```

Comment:
Why did we remove the name item in the discovery info dict? That was setting the name of the notify service.